lib/hanami/cli/commands/command.rb

```diff
@@ -160,7 +175,26 @@ def generate_file(source, destination, context)
         # @since 1.1.0
         # @api private
         def say(operation, path)
-          out.puts(SAY_FORMATTER % { operation: operation, path: path }) # rubocop:disable Style/FormatString
+          out.puts(_format_say(operation, path))
+        end
+
+        # @since x.x.x
+        # @api private
+        def _format_say(operation, path)
+        [
+          _colorize_operation(operation),
+          path
+        ].join("  ")
+        end
+
+        # @since x.x.x
+        # @api private
+        def _colorize_operation(operation)
+          if OPERATION_COLORS.key?(operation)
+            _colorize_and_justify(operation)
+          else
+            operation.to_s.rjust(COLUMN_WIDTH)
```

Comment:
We _could_ just colorize other stuff to black, to get rid of this conditional.

Reply:
@cllns This assumes a black terminal background.